modules/cluster-logging-exported-fields-top-level-fields.adoc

```diff
@@ -10,38 +9,52 @@
 
 The top level fields may be present in every record.
 
-=== message
-
+== message
 
 Original log entry text, UTF-8 encoded This field may be absent or empty if a non-empty `structured` field is present. See the description of `structured` for more.
```

Comment:
An article and a full stop missing?
   An/the original log entry text, UTF-8 encoded. This field may be....

Reply:
Updated. Thanks.